lemur/authorities/views.py

```diff
@@ -300,6 +302,7 @@ def get(self, authority_id):
         return service.get(authority_id)
 
     @validate_schema(authority_update_schema, authority_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
this already requires role admin so it shouldn't be necessary https://github.com/Netflix/lemur/blob/1c096f1028a237a6f58d181e5aaabb0f8f1e0953/lemur/auth/permissions.py#L80